server/adaptors/index.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+import { SELECTED_BACKEND } from '../../common';
+import { ZeppelinBackend } from './zeppelin_backend';
+import { DefaultBackend } from './default_backend';
+import { NotebookAdaptor } from './notebook_adaptor';
+
+let BACKEND: NotebookAdaptor;
+
+// Selects backend based on config
+if (SELECTED_BACKEND == 'ZEPPELIN') {
```

Comment:
Can simplify this- initialize `BACKEND` to `new DefaultBackend()` and then have `if` block if `SELECTED_BACKEND == 'ZEPPELIN'`

Reply:
Done! 👍